projects/librdkafka.cmake

```diff
@@ -1,13 +1,13 @@
 ########################################
-# qt5
+# librdkafka
 ########################################
 xpProOption(librdkafka)
-set(KAFKA_VER 0.9.0)
+set(KAFKA_VER master)
 set(KAFKA_REPO https://github.com/edenhill/librdkafka)
```

Comment:
I updated the distributePro form and created a "pre-release" tag of 16.03.1 and switched the librdkafka.cmake file to use that tag when building. Also updated the readme to reflect that's the version we're using.